docs/4-user-guide/1-the-zarf-cli/100-cli-commands/zarf_completion.md

```diff
@@ -30,4 +30,4 @@ See each sub-command's help for details on how to use the generated script.
 * [zarf completion powershell](zarf_completion_powershell.md)	 - Generate the autocompletion script for powershell
 * [zarf completion zsh](zarf_completion_zsh.md)	 - Generate the autocompletion script for zsh
 
-###### Auto generated by spf13/cobra on 17-Jul-2022
+###### Auto generated by spf13/cobra on 25-Jul-2022
```

Comment:
I do wish there was a way to avoid triggering updates on every page--even ones without an actual update 😭